proxy/src/main/java/com/velocitypowered/proxy/VelocityServer.java

```diff
@@ -605,11 +606,8 @@ public InetSocketAddress getBoundAddress() {
   }
 
   @Override
-  public void sendMessage(net.kyori.adventure.text.@NonNull Component message) {
-    Preconditions.checkNotNull(message, "message");
-    for (ConnectedPlayer player : connectionsByUuid.values()) {
-      player.sendMessage(message);
-    }
+  public @NonNull Iterable<? extends Audience> audiences() {
+    return this.getAllPlayers();
   }
```

Comment:
Changes list says `ProxyServer` applies actions to all players including console, does this not just give the player audiences?

Reply:
Should be fixed now.